public/app/features/alerting/unified/components/rule-editor/alert-rule-form/simplifiedRouting/AlertManagerRouting.tsx

```diff
@@ -71,14 +82,14 @@ export function AlertManagerManualRouting({ alertManager }: AlertManagerManualRo
   );
 }
 function LinkToContactPoints() {
+  const styles = useStyles2(getStyles);
   const hrefToContactPoints = '/alerting/notifications';
   return (
     <Link target="_blank" href={createUrl(hrefToContactPoints)} rel="noopener" aria-label="View alert rule">
-      <Stack direction="row" gap={1} alignItems="center" justifyContent="center">
-        <Text color="secondary">To browse contact points and create new ones, go to</Text>
-        <Text color="link">Contact points</Text>
+      <div className={styles.contactPointsInfo}>
+        <Text color="link">View or create contact points</Text>
         <Icon name={'external-link-alt'} size="sm" color="link" />
-      </Stack>
+      </div>
```

Comment:
I removed the label and updated the text to make it more clear. I have a confirmation from @brendamuir in our slack channel.

<img width="673" alt="Screenshot 2024-01-18 at 10 47 12" src="https://github.com/grafana/grafana/assets/33540275/a7a49862-6d72-4552-b9f6-738f1e021576">

Reply:
The design system has a `TextLink` component we should use instead – which applies the correct color to the icon too :) 

```
<TextLink external href={createUrl(hrefToContactPoints)} aria-label="Go to contact points">
  View or create contact points
</TextLink>
```